uhubctl.c

```diff
@@ -1051,9 +1058,12 @@ int main(int argc, char *argv[])
                 for (port=1; port <= hubs[i].nports; port++) {
                     if ((1 << (port-1)) & ports) {
                         int port_status = get_port_status(devh, port);
+                        printf("Port Status: %i\n", port_status);
```

Comment:
We don't want to always print port status here. Is this left from your debugging?

Reply:
Of course - yes.